includes/functions/llms-functions-content.php

```diff
@@ -112,6 +112,24 @@ function llms_get_post_sales_page_content( $post, $default = '' ) {
 
 }
 
+/**
+ * Retrieve the decimal value for average column in Reporting.
+ *
+ * By default the database return floating-point values and  function
+ * percision to round off the decimal value.
+ *
+ * @since [version]
+ *
+ * @return int
+ */
+function llms_get_reporting_round_precision() {
+
+	$precision = apply_filters( 'lifterlms_average_decimal', 1 );
```

Comment:
I would use a more contextualized yet less specific (average? can't we use it in general?) hook name. Dunno, something like:

`lifterlms_reporting_floats_rounding_precision`

or something of the sort.
What do you think?

Also, could you please add a docblock for this new filter?

What do you think if we run this filter once? Like only the first time we call this function?

Reply:
done! thanks 👍